AccessCheckoutSDK/AccessCheckoutSDKPactTests/AccessCheckoutSDKPactTests.swift

```diff
@@ -13,13 +13,20 @@ class AccessCheckoutSDKPactTests: XCTestCase {
                                                 consumer: "access-checkout-iOS-sdk")
 
     class MockDiscovery: Discovery {
+        var serviceEndpoint: URL?
+        
         var verifiedTokensSessionEndpoint: URL?
+        var sessionsEndpoint: URL?
+        /// Starts discovery of services
         let baseURI: String
         init(baseURI: String) {
             self.baseURI = baseURI
         }
-        func discover(urlSession: URLSession, onComplete: (() -> Void)?) {
+        
+        func discover(serviceLinks: DiscoverLinks, urlSession: URLSession, onComplete: (() -> Void)?){
+            serviceEndpoint = URL(string: "\(baseURI)/verifiedTokens/sessionssss")
             verifiedTokensSessionEndpoint = URL(string: "\(baseURI)/verifiedTokens/sessions")
+            sessionsEndpoint = URL(string: "\(baseURI)/sessions/payments/cvc")
```

Comment:
Question for you - Do we need to add the sessionsEndpoint? Or is that something we will do as part of the changes to the PACT contract?